4.0/Makefile

```diff
@@ -0,0 +1,77 @@
+BUILDDIR=build
+TOOLSDIR=tools
+SOURCEDIR=en
+TEMPLATEDIR=templates
+SOURCES = $(SOURCEDIR)/*.md
+TARGETS=$(addprefix $(BUILDDIR)/,$(wildcard $(SOURCES)))
+TARGETNAME=OWASP Application Security Verification Standard 4.x
+#EXPORT_JSON_TOOL=$(TOOLSDIR)/generate-json.py
+EXPORT_TOOL=$(TOOLSDIR)/export.py
+#EXPORT_CSV_TOOL=$(TOOLSDIR)/generate-csv.py
+
+# Change LaTeX engine
+PANDOC_MD_FLAGS=-f gfm -s -t markdown
+
+PANDOC_PDF_FLAGS=-f markdown -s -t latex --pdf-engine=xelatex 
+
+PANDOC_TEX_FLAGS=-f markdown -s -t latex 
+
+PANDOC_DOCX_FLAGS= -s \
+				  -t docx \
+				  -f markdown \
+				  --toc \
+				  --columns 10000 \
+				  --reference-doc=./templates/reference.docx 
+
+PANDOC_ODT_FLAGS= -s \
+				  -t odt \
+				  -f markdown \
+				  --toc \
+				  --columns 10000 \
+				  --reference-doc=./templates/reference.odt
+
+.PHONY: md $(BUILDDIR) pdf docx odt clean
+
+all: $(TARGETS)	pdf docx json json_flat csv xml
+
+$(BUILDDIR): 
+	@mkdir -p $(BUILDDIR)/en # make sure build dir exists
+
+$(BUILDDIR)/en/%.md: $(BUILDDIR)
+	sed -E 's#(\| ?)([0-9]{1,4})( ?\|)#\1[\2](https://cwe.mitre.org/data/definitions/\2.html)\3#; s#^(\| :?---:? \| :?)---( .*)#\| :-----: \| :---------------------------------------------------\2#; s#.(./images/)#\1#' $(subst build,.,$@) > $@
+	 
+md: $(BUILDDIR)
+	pandoc $(PANDOC_MD_FLAGS) -o $(BUILDDIR)/source.md $(BUILDDIR)/$(SOURCES) 
+	sed -Ei 's#../images/#./images/#' $(BUILDDIR)/source.md
+
+pdf: 
+	pandoc $(PANDOC_PDF_FLAGS) --include-in-header=$(TEMPLATEDIR)/header.tex  -o "$(BUILDDIR)/$(TARGETNAME).pdf" $(BUILDDIR)/en/*.md $(SOURCEDIR)/0x00-Header.yaml
+	pandoc $(PANDOC_PDF_FLAGS) --include-in-header=$(TEMPLATEDIR)/header-eisvogel.tex -o "$(BUILDDIR)/$(TARGETNAME)2.pdf"  --template $(TEMPLATEDIR)/eisvogel.tex $(BUILDDIR)/en/*.md $(SOURCEDIR)/0x01-Header.yaml
```

Comment:
Will these two templates eventually converge? If not, we might want to consider a clearer filename for the second -- "2" -- file.

Reply:
I would rather delete the one not using Eisvogel. No need to keep it. The decision has been taken